packages/appstore/src/assets/svgs/wallets/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import UpgradeDesktop from 'Assets/svgs/wallets/wallets-upgrade-desktop.svg';
+import UpgradeMobile from 'Assets/svgs/wallets/wallets-upgrade-mobile.svg';
+
+export const WalletsImagesList = {
+    UpgradeDesktop,
+    UpgradeMobile,
+};
+
+type WalletsImageProps<T> = {
+    banner: T;
+    className?: string;
+};
+
+const WalletsImage = ({ banner, className }: WalletsImageProps<keyof typeof WalletsImagesList>) => {
```

Comment:
@mahdiyeh-deriv in my opinion it's not icon, it's image, because these images are not clickable and size is not square. I saw examples for `trading-platform` and `currency` icons, but I think it's another case, it's not an icon. But it's just my suggestion, I'll do so as you say :) Let me know your final decision